### ISSUE TITLE
Java Runtime Installation

### ISSUE THREAD

**Issue by akilegaspi** [d2rm/d2rm#26]
I already have the lastest JRE installed but the application won't install because it cannot detect the Java Runtime.

Does it look for a specific environment variable?

**Comment by kleinfreund** [comment]
Indeed. I have set `C:\Program Files\Java\jre1.8.0_77\bin` in `PATH` under the user-level environment variables on Windows but the installer does not recognize it is installed.
